reflex/state.py

```diff
@@ -1532,14 +1539,22 @@ async def _process_event(
                 yield state._as_state_update(handler, events, final=True)
 
         # If an error occurs, throw a window alert.
-        except Exception:
-            error = traceback.format_exc()
-            print(error)
-            yield state._as_state_update(
-                handler,
-                window_alert("An error occurred. See logs for details."),
-                final=True,
-            )
+        except Exception as e:
+            stack_trace = traceback.format_exc()
+
+            config = get_config()
+
+            # Call the custom backend exception handler if specified. If not, fallback to the default handler.
+            if config.backend_exception_handler:
```

Comment:
IMO it's cleaner to drop this logic here and just set the default value for `config.backend_exception_handler` to `default_backend_exception_handler`